releases/latest/edge/bundle-airgap.yaml

```diff
@@ -0,0 +1,261 @@
+bundle: kubernetes
+name: kubeflow
+applications:
+  admission-webhook:
+    charm: ./admission-webhook_98aac65.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/kubeflownotebookswg/poddefaults-webhook:v1.7.0
+  argo-controller:
+    charm: ./argo-controller_b59eaec.charm
+    scale: 1
+    resources:
+      oci-image: 172.17.0.2:5000/argoproj/workflow-controller:v3.3.8
+    options:
+      executor-image: 172.17.0.2:5000/argoproj/argoexec:v3.3.8
+  argo-server:
+    charm: ./argo-server_6d22972.charm
+    scale: 1
+    resources:
+      oci-image: 172.17.0.2:5000/argoproj/argocli:v3.3.8
+  dex-auth:
+    charm: ./dex-auth_f0211e2.charm
+    scale: 1
+    trust: true
+    resources:
+      oci-image: 172.17.0.2:5000/dexidp/dex:v2.36.0
+  istio-ingressgateway:
+    charm: ./istio-gateway_ubuntu-20.04-amd64.charm
+    scale: 1
+    trust: true
+    options:
+      kind: ingress
+      proxy-image: 172.17.0.2:5000/istio/proxyv2:1.17.3
+      gateway_service_type: NodePort
+  istio-pilot:
+    charm: ./istio-pilot_ubuntu-20.04-amd64.charm
+    scale: 1
+    trust: true
+    options:
+      default-gateway: kubeflow-gateway
+      image-configuration: '{"pilot-image": pilot, "global-tag": 1.17.3, "global-hub": 172.17.0.2:5000/istio, "global-proxy-image": proxyv2, "global-proxy-init-image": proxyv2, "grpc-bootstrap-init": busybox:1.28}'
```

Comment:
Is there a chance this needs for the values in the dictionary `""`? This is more a question, I just wanted to raise. Probably it is ok like this also

Reply:
No I meant that the values don't even have a `"` e.g. `"pilot-image": pilot`

Reply:
Yes. Better to have those. I will address it.

Reply:
I'd say to be on the safe side, let's use `"` on all values